### DISCUSSION TITLE
Not able to download atom from the official page

### DISCUSSION THREAD

**Discussion by maybearun** [atom/atom#22336] [Q&A] [answered]
Hello,
I am not able to download atom from the official page. The page gets stuck or just keeps loading when clicked on the download page. 

Please direct me on how to download the official release of Atom.

Thank You

**Comment by tiredkangaroo** [comment]
The official release of atom is on https://atom.io Make sure to left click the download button and a save button will pop-up. Save it and run the file. If it is stuck loading, check your internet bandwidth. Which OS and browser are you on?

**Reply by maybearun** [reply]
I'm using a windows device i tried it on chrome as well as Firefox but on clicking the download button the site just loads till the browser puts up the prompt "atom-installer.github.com took too long to respond."

![Screenshot (255)](https://user-images.githubusercontent.com/58701858/116896262-b097d100-ac51-11eb-963f-bd185c54dc0b.png)

**Comment by tiredkangaroo** [comment, marked as answer]
try this: https://github.com/atom/atom/releases/download/v1.56.0/AtomSetup-x64.exe

**Reply by maybearun** [reply]
Thank you it worked
